### PR TITLE
Hyperledger Aries 2024 Q3 Quarterly Report

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: project-reports/2024/2024-Q3-Hyperledger-Aries.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+## Project Health
+
+Work on [Hyperledger Aries] continues at a healthy pace, with lots of activity
+in all of the Aries and the related-OWF sub-communities. [Aries Cloud Agent
+Python] continues to move to its 1.0.0 release, and the community around
+[Aries-VCX] is again growing, with lots of work happening in moving the
+implementation forward. [Open Wallet Foundation]-hosted [Credo-TS] and [Bifold
+Wallet] continue to evolve rapidly. [Animo Solutions], key maintainers of
+[Credo-TS], were selected by the German Government as funded participate in
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Aries.md]: something not quite right with the grammar here.

File: project-reports/2024/2024-Q3-Hyperledger-Aries.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+obsolete (but unchanging) repositories. Per Ry Jones there is not a good way for
+us to control the included list of repos.
+
+[Aries Activity Dashboard]: https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=aries&routedFrom=Github&bestPractice=false&dateFilters=Last%20Quarter&dateRange=2024-04-01%20to%202024-06-30&compare=PP&granularity=week&hideBots=true
+
+- The [Indy SDK] has been removed from ACA-Py, eliminating the older component in favour of newer and more capable
+  replacements ([Aries Askar], [Indy VDR], and [AnonCreds RS]). Good to see that
+  transition!
+  - **NOTE: THIS DOES NOT REMOVE INDY SUPPORT FROM ACA-PY!**
+- Support for AnonCreds in W3C VCDM Format in Aries has been implemented in [Aries Cloud Agent Python].
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Aries.md:67]:
```suggestion
- Support for AnonCreds in W3C VCDM Format in Aries has been implemented in [Aries Cloud Agent Python plugins].
```

(at least to match the link on line 72)

File: project-reports/2024/2024-Q3-Hyperledger-Aries.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+- Work has begun on adding [DIDComm v2] support in [ACA-Py].
+
+[Aries Askar]: https://github.com/hyperledger/aries-askar
+[Indy SDK]: https://github.com/hyperledger/indy-sdk
+[Hyperledger AnonCreds]: https://github.com/hyperledger/anoncreds-rs
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Aries.md]:
```suggestion
```

not used

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Minor updates requested, but looks good otherwise.

File: project-reports/2024/2024-Q3-Hyperledger-Aries.md (hunk 1 above)
Comment (swcurran) [reply project-reports/2024/2024-Q3-Hyperledger-Aries.md]: Reworded.

File: project-reports/2024/2024-Q3-Hyperledger-Aries.md (hunk 2 above)
Comment (swcurran) [reply project-reports/2024/2024-Q3-Hyperledger-Aries.md:67]: The link was correct. The plugins link was not used and the definition removed.

File: project-reports/2024/2024-Q3-Hyperledger-Aries.md [hunk 4]
```diff
@@ -16,16 +16,17 @@ Python] continues to move to its 1.0.0 release, and the community around
 [Aries-VCX] is again growing, with lots of work happening in moving the
 implementation forward. [Open Wallet Foundation]-hosted [Credo-TS] and [Bifold
 Wallet] continue to evolve rapidly. [Animo Solutions], key maintainers of
-[Credo-TS], were selected by the German Government as funded participate in
-their [Funke] ("sparkle") initiative to build innovation in the EUDI space.
+[Credo-TS], were selected by the German Government as a funded participate in
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Aries.md]:
```suggestion
[Credo-TS], were selected by the German Government as a funded participant in
```

**Review by tkuhrt — APPROVED** [review]

**Review by petermetz — APPROVED** [review]

**Review by arsulegai — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Aries.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+VCX and Aries Cloud Agent Python (ACA-Py) within DID Exchange, adding support
+for DID Exchange 1.1 and did:peer numalgo 4.
+
+Release 0.65.0 marks the planned final release including support for the
+indy-sdk, credx, and indyvdrtools crates and features in Aries VCX. The
+replacement for these crates are already implemented (Aries Askar, anoncreds-rs,
+and indy-vdr) in Aries VCX. Additional info or discussion can be found here.
+Work is underway to remove these dependencies from the project.
+
+Work is in-progress on fixing and enhancing fixing Android support, as well as
```
Comment (jimthematrix) [inline project-reports/2024/2024-Q3-Hyperledger-Aries.md:96]: Would be good to have at the top of this section what is VCX and how it's different from the Bifold sub-project (my understanding is both are client side SDK targeted at mobile app development?).

**Review by jimthematrix — APPROVED** [review]
approved with a minor comments on clarity

File: project-reports/2024/2024-Q3-Hyperledger-Aries.md (hunk 4 above)
Comment (swcurran) [reply project-reports/2024/2024-Q3-Hyperledger-Aries.md]: Fixed

**Review by mbrandenburger — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**